File: python/frgtnlng.py
T = int(input())
is_old_word = []

def getIsOld(old_words, new_words):
    is_old = []
    for word in old_words:
        is_old.append("YES") if word in new_words else is_old.append("NO")
    return is_old

for _ in range(T):
    old_word_cnt, new_sentence_cnt = map(int, input().split())
    old_words = input().split()
    new_words = []
    for sentence_num in range(new_sentence_cnt):
        new_words = new_words + input().split()
    is_old_word.append(getIsOld(old_words, new_words))

for res in is_old_word:
    print(res)

File: java/FrgtnLng.java
import java.util.Scanner;
class FrgtnLng {
	int newWordCount;
	int oldWordCount;
	public final String PRESENT = "YES";
	public final String ABSENT = "NO";
	public Frgtnlng() {
	}
	public void setValues(int newWordCount,	int oldWordCount ) {
		this.oldWordCount = oldWordCount;
		this.newWordCount = newWordCount;
	}
	public String getIsOld(String oldWords[], String newWords[]) {
		String word = ABSENT;
		return word;
	}
	public static void main(String args[]) {
		Scanner s = new Scanner(System.in);
		int T = s.nextInt();
		FrgtnLng lng = new FrgtnLng();
	}
}
